src/iot/feedTypes.js

```diff
@@ -0,0 +1,57 @@
+import { toCamelCase } from '../utils/objects';
+
+/**
+ * @typedef {Object} FeedType
+ * @property {String} createdAt
+ * @property {String} downAfter
+ * @property {String} id UUID
+ * @property {String} troubleshootingUrl
+ * @property {String} type
```

Comment:
Can this be anything?

Reply:
I've added a comment. It's actually the name of the feed.